src/components/redux/reducers/index.jsx

```diff
@@ -1,16 +1,53 @@
 import { combineReducers } from "redux";
+import {
+  ADD_TO_CART,
+  DELETE_FROM_CART,
+  MODIFY_QUANTITY,
+  SUM_TOTAL_PRICE,
+  ADD_ONE_MORE_ITEM,
+  REMOVE_ONE_MORE_ITEM,
+  QUANTITY_IN_SINGLE_PRODUCT,
+} from "../actions/actionsTypes";
 
-const currentBoardIDReducer = (state = [], action) => {
+const addOrDeleteProductData = (state = [], action) => {
   switch (action.type) {
-    case "CURRENT_BOARD_ID":
+    case ADD_TO_CART:
+      return [...state, action.payload];
+    case DELETE_FROM_CART:
+      return action.payload;
+    case MODIFY_QUANTITY:
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+const getTotalPrice = (state = [], action) => {
```

Comment:
hmm...this function confuses me. its called getTotalPrice so i would expect it returns a number, but line 28 returns an array. perhaps need some comments or a rename